### ISSUE TITLE
Add `Edit on Github` button to tutorials

### ISSUE THREAD

**Issue by tmigot** [JuliaSmoothOptimizers/JuliaSmoothOptimizers.github.io#83]
That's a feature encouraging contributors to edit tutorials.

**Comment by abelsiqueira** [comment]
Clarifying this issue:

- The tutorials are stored in JSOTutorials.jl
- The edit button should be on the individual tutorial page
- Clicking the button should lead to the JSOTutorials.jl file for that specific tutorial

**Comment by Jay-sanjay** [comment]
@abelsiqueira @dpo Wanted to be clear that for example I want to edit tutorials of this -https://jso.dev/tutorials/advanced-jsosolvers/ then I should point to this -https://github.com/JuliaSmoothOptimizers/JSOTutorials.jl/tree/main/tutorials/advanced-jsosolvers for editing. Please correct me if I am wrong.

**Comment by abelsiqueira** [comment]
@Jay-sanjay, that is correct. Possibly in the weaving (in JSOTutorials.jl), so it is automatically modified whenever we add new tutorials.
